responsibleai/responsibleai/modelanalysis/model_analysis.py

```diff
@@ -309,6 +309,13 @@ def _validate_model_analysis_input_parameters(
                             ' getting predictions via predict_proba()'
                         )
 
+                if task_type == ModelTask.REGRESSION:
```

Comment:
Might consider making this an `elif` of the condition above. Otherwise looks good

Reply:
Keeping this as a spearate if.

Reply:
nit: I think using and is better than two ifs, but please ignore if you prefer this way

```if task_type == ModelTask.REGRESSION and hasattr(model, PREDICT_PROBA):```